governance-exercises/DAO Masons poll-exercise-04.md

```diff
@@ -0,0 +1,58 @@
+# Poll 4
+
+### Determine what course of action you would take as a Governance Facilitator and communicate it clearly.
+
+Protocol Scope Facilitator Imogen has requested an urgent poll to gather MKR sentiment on if a costly security fix should go live.
+
+Our read on the situation:
+Fix B Executive is going up regardless, very shortly.
+Fix A Executive will be ready around the same time and may go up based on MKR holder sentiment.
+A Technical Ecosystem Actor and Protocol Scope Facilitator are requesting a poll to see if the DAO supports Fix A, which will likely result in a 20mm loss to MKR holders.
+
+I would start a chat with Geneveive and/or Imogen and verify the timeline, and ask them questions publicly on the forum as well. I'm going to pretend that they said the fixes would be ready in 8 hours.
+
+During this conversation I'm going to propose a different course of action rather than creating a poll about Fix A implementation. If they make an executive proposal for Fix A and Fix B, then they can go up at the same time and start their voting periods. If they both pass, then the DAO has spoken and both are executed. If only Fix B passes, then that's that and we are left vulnerable for 2 weeks.
+
+I could then spend the remaining time waiting for the execs to be ready 'coordinating an emergency governance call to discuss solutions and a plan for subsequent actions, explicitly inviting known stakeholders' as dictated by MIP24c3 [Emergency Response Procedure](https://mips.makerdao.com/mips/details/MIP24#MIP24c3).
+
+This reduces the time that we're signaling to the world that there is a vulnerability in Maker and reduces risk. They mentioned that the exploit can't be used in 'current circumstances' but it's unclear how dynamic those circumstances are and it's possible we could be attacked if we signal we are vulnerable. Technically the vulnerability is already signaled by the forum post so avoiding the poll only reducing the broadcast.
```

Comment:
Yes. In practice technical teams would likely be even more cagey about releasing anything in a situation like this. Makes it harder to do an exercise about it though.